osquery/bro/BrokerManager.cpp

```diff
@@ -0,0 +1,312 @@
+#include <osquery/flags.h>
```

Comment:
We should follow the:
1. C includes
2. C++ includes
3. Broad project dependencies includes
4. File-specific dependency includes
5. osquery includes
6. osquery local includes

Order of includes, keeps us honest and organized.